nck/helpers/mytarget_helper.py

```diff
@@ -1,4 +1,5 @@
-from string import Template
+
+REQUEST_TYPES = ["general", "budget"]
```

Comment:
I think "performance" would have been a better name than "general". This is closer to the names used by the business.

Reply:
Np, will do right away